### PR TITLE
NEW Add scaffolding

### PR DESCRIPTION
**Pull request by raissanorth**
- Add very basic folder structure
- Add linting and test configuration
- Add content to `README` file

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  "author": "SilverStripe Ltd",
+  "license": "BSD-3-Clause",
+  "bugs": {
+    "url": "https://github.com/silverstripe/login-forms/issues"
+  },
+  "homepage": "https://github.com/silverstripe/login-forms#readme",
+  "dependencies": {
+  },
+  "devDependencies": {
+    "@silverstripe/webpack-config": "^0.4.0",
```
Comment (ScopeyNZ) [inline package.json]: I think we're up to `^0.14.0`.

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        "silverstripe",
+        "template",
+        "login",
+        "styling"
+    ],
+    "require": {
+        "silverstripe/framework": "^4"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^5.7",
```
Comment (Firesphere) [inline composer.json]: Please add `"roave/security-advisories": "dev-master"`?

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    "keywords": [
+        "silverstripe",
+        "template",
+        "login",
+        "styling"
+    ],
+    "require": {
+        "silverstripe/framework": "^4"
+    },
+    "require-dev": {
```
Comment (Firesphere) [inline composer.json]: Can we add `"scriptfusion/phpunit-immediate-exception-printer"`? When the output doesn't run in to the hundreds of tests, it's very useful and visual:
https://circleci.com/gh/Firesphere/silverstripe-bootstrapmfa/117

File: package.json (hunk 1 above)
Comment (raissanorth) [reply package.json]: Of course we are, good spotting! Will update...

File: composer.json (hunk 2 above)
Comment (robbieaverill) [reply composer.json]: Yep, this is a PHPStorm/EA Inspections suggestion which we haven't standardised on yet

File: composer.json (hunk 3 above)
Comment (robbieaverill) [reply composer.json]: We generally try not to introduce any dependencies that aren't part of our standardised modules - if you'd like this to be introduced I think it'd be best to raise an RFC on silverstripe/framework

Comment (raissanorth) [reply composer.json]: @ScopeyNZ @robbieaverill @NightJar thoughts?

Comment (Firesphere) [reply composer.json]: I do locally ;) Which Is why I add it to my repos

File: composer.json (hunk 2 above)
Comment (robbieaverill) [reply composer.json]: Sorry I misread this, same comment for this as the other - please don’t add anything that isn’t part of our standardised set already without an RFC and widespread adoption

File: package.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "login-forms",
+  "version": "1.0.0",
```
Comment (NightJar) [inline package.json]: I feel like this should be a sub-1.0.0 note for now. Perhaps `0.0.1`?

Comment (NightJar) [reply package.json]: Wait... this isn't composer - how does package.json versioning work for something that is never 'published'? Is it even necessary?

Comment (raissanorth) [reply package.json]: https://docs.npmjs.com/files/package.json#version:
> If you don't plan to publish your package, the name and version fields are optional.

File: .editorconfig [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# For more information about the properties used in this file,
+    # please see the EditorConfig documentation:
+    # http://editorconfig.org
+
+    [*]
+    charset = utf-8
+    end_of_line = lf
```
[the remark this answers is not in the document]
Comment (raissanorth) [reply .editorconfig]: I could pretend there is ... but nah! 😉

**Pull request merged**